processor/processor.go

```diff
@@ -1981,6 +1984,10 @@ func (proc *Handle) Store(partition string, in *storeMessage) {
 				proc.reporting.Report(in.reportMetrics, tx.SqlTx())
 			}
 
+			if proc.isErrorReportingEnabled() {
+				proc.errorReporting.Report(in.reportMetrics, tx.SqlTx())
+			}
```

Comment:
Can we use Noop implementation instead of this if check?

Reply:
Do we have an example snippet for this in our code-base ? If yes, can you share it ?

Reply:
A flavor of NOOP implementation is done. Kindly check and let me know if this is what you wanted.

Reply:
This is already getting handled as part of `reporting.go` `Setup` function, where in we are setting the `NOOP` while initialising itself. Handled it that way